icon_validator/rules/plugin_validators/confidential_validator.py

```diff
@@ -25,6 +26,7 @@ def validate_help(plugin_path: str):
     # Check content line by line for emails that validate the rule
     @staticmethod
     def validate_emails(content: [str], path_to_file: str):
+        print(content)
```

Comment:
Should we remove this print?

Reply:
Yep! Totally left it there by mistake lol